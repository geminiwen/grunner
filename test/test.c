#include <stdio.h>
#include <stdlib.h>
#include <unistd.h>
#include <sys/fcntl.h>

int main() {
    int a,b;

    int aa = open("/Users/geminiwen/Code/CLionProjects/grunner/test/test.sample2", O_CREAT);
    int ab = open("/tmp/test.txt", O_RDONLY);
    printf("aa:%d\n", aa);
    printf("ab:%d\n", ab);
    return 0;
}