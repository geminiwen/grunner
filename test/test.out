aa:-1
ab:-1
